letsencrypt/cli.py

```diff
@@ -185,15 +218,86 @@ def run(args, config, plugins):
         return "Configurator could not be determined"
 
     domains = _find_domains(args, installer)
+
```

Comment:
I feel like this whole block of code can go in a function so that auth can simply call it